src/fu-util.c

```diff
@@ -2792,7 +2883,7 @@ main (int argc, char *argv[])
 			target_fname = g_steal_pointer (&log);
 		}
 		file = g_file_new_for_commandline_arg (target_fname);
-		log_output = g_file_append_to (file, G_FILE_CREATE_NONE, NULL, &error);
+		log_output = g_file_replace (file, NULL, FALSE, G_FILE_CREATE_NONE, NULL, &error);
```

Comment:
If this is something that's only used my MOTD, should we specify the MOTD file for get-motd and just set it there? If it's useful for other things it can stay of course.

Reply:
Hmm, that's a good point.  It was done in a generic way to work for anything, but given it's just a single string now, we should question that.

Actually I have to question does motd update command even make sense as a client feature?  Maybe we should be having the **daemon** update motd file every time it gets a refresh cycle?

Reply:
> given it's just a single string now

Indeed.

>  Maybe we should be having the daemon update motd file

Hmmf. I could argue this either way. My hunch is the client side way is better, otherwise we're just jamming in stuff that doesn't need to be run as the root user.

Reply:
The argument for moving to daemon is that the motd will always reflect the latest state of the devices the daemon knows. So on every install, refresh, hotplug etc it can be updated.

Reply:
Maybe do a quick prototype and see how it works in the daemon?

Reply:
OK, pushed an extra commit that roughly does it in the daemon.  See what you think of that.
The times it's called might need some work (advice welcome if you think this is correct direction to go).

If it's the right direction i'll adjust this PR to tear out all the client stuff.